content/blog/2022/2022-08-25-schedule-cron-job-in-node.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Schedule Job On Node.js"
+categories: ["Node"]
+date: 2022-08-25 00:00:00 +1100
+authors: ["ajibade"]
+description: "Quick guide to scheduling cron jobs in Node.js applications"
+image: images/stock/0031-matrix-1200x628-branded.jpg
+url: schedule-cron-job-in-node
+---
+
+Have you ever wanted to perform a specific task on your application server at specific times without physically running them yourself? Or you'd rather spend your time on more important tasks than remember to periodically clear or move data from one part of the server to another. Cron Job schedulers can be used to achieve this easily.
+
+Cron job scheduling is a common practice in modern applications; it provides a level of autonomy in our applications. Cron is a [daemon](https://en.wikipedia.org/wiki/Daemon_(computing)), which means that it only needs to be started once, and will 
+lay dormant until it is required. Another example of a deamon is the web server. The web server remains idle until a request for a web page is made. 
+
+## Using Node Cron in Node.Js
+In this article, we'll use the [node-cron](https://www.npmjs.com/package/node-cron) library to schedule cron jobs in node.js and build several demo applications. Node-cron is a node module that is made available by npm.
+
+To schedule jobs using `node-cron`, you need to invoke the cron `.schedule` method.
+
+### Cron Schedule Method Syntax
+```
+cron.schedule(expression , function, options);
+```
+As shown above, the cron `.schedule` method takes in three arguments where :
+
+#### The First Argument (Expression)
```

Comment:
The header shouldn't be "The First Argument", but "The Cron Expression" to give more context. In the first sentence of the text, you can make the connection to the first argument